.changeset/strong-rice-warn.md

```diff
@@ -0,0 +1,35 @@
+---
+'@backstage/core-components': patch
+'@backstage/core-plugin-api': patch
```

Comment:
To be honest I actually got a bit confused here myself about what changes belong to `core-components` and which ones belong to `core-plugin-api`. I'd say it's best to split this into two changesets, where the `core-plugin-api` one probably doesn't mention the `AlertDisplay` at all. It's likely best that the `core-components` one mentions that this is for the new `AlertMessage` option, but also be sure to mention these messages are tied to the `AlertApi`.

Reply:
Ok, I've split the changeset into two, hopefully I've got things in the right place